woocommerce/woocommerce-admin
# make store revenue range amounts translatable

Fixes #2768

This PR adds translation & translation instructions for the revenue amount ranges on the business details screen of onboarding.

### Detailed test instructions:

- Reset onboarding with `wp option delete wc_onboarding_profile`
- Enable onboarding in wp-config.php with `WOOCOMMERCE_ADMIN_ONBOARDING_ENABLED`
- Navigate through the onboarding flow to the business details
- Select one of the physical location options
- Choose one of the revenue amounts
- Continue
- Check the saved value `wp option get wc_onboarding_profile`

### Changelog Note:

Fix: Enable translation of currency amounts for business details.
